ako-crd-operator/config/crd/bases/ako.vmware.com_healthmonitors.yaml

```diff
@@ -0,0 +1,232 @@
+                description: MonitorPort is the port to use for the health check.
+                format: int32
+                maximum: 65535
+                minimum: 0
+                type: integer
+              receive_timeout:
+                default: 4
+                description: ReceiveTimeout is the timeout for receiving a ping response,
+                  in seconds.
+                format: int32
+                maximum: 2400
+                minimum: 1
+                type: integer
+              send_interval:
+                default: 10
+                description: SendInterval is the frequency, in seconds, that pings
+                  are sent.
+                format: int32
+                maximum: 3600
+                minimum: 1
+                type: integer
+              successful_checks:
+                default: 2
+                description: SuccessfulChecks is the number of successful pings before
+                  marking up.
+                format: int32
+                maximum: 50
+                minimum: 1
+                type: integer
+              tcp:
+                description: TCP defines the TCP monitor configuration.
+                properties:
+                  maintenance_response:
+                    description: MaintenanceResponse is the response to send when
+                      in maintenance mode.
+                    maxLength: 512
+                    type: string
+                  tcp_half_open:
+                    description: TcpHalfOpen is a boolean to check if the tcp monitor
+                      is in half open mode
+                    type: boolean
+                  tcp_request:
+                    description: TcpRequest is the request to send for the TCP health
+                      check.
+                    maxLength: 1024
+                    type: string
+                  tcp_response:
+                    description: TcpResponse is the expected response for the TCP
+                      health check.
+                    maxLength: 512
+                    type: string
+                type: object
+                x-kubernetes-validations:
+                - message: tcp_request, tcp_response, and maintenance_response cannot
+                    be set when tcp_half_open is true
+                  rule: self.tcp_half_open == false || (self.tcp_request == '' &&
+                    self.tcp_response == '' && self.maintenance_response == '')
+              type:
+                description: Type is the type of health monitor.
+                enum:
+                - HEALTH_MONITOR_TCP
+                - HEALTH_MONITOR_PING
+                - HEALTH_MONITOR_HTTP
+                - HEALTH_MONITOR_HTTPS
+                type: string
+                x-kubernetes-validations:
+                - message: type is immutable
+                  rule: self == oldSelf
+            required:
+            - type
+            type: object
+          status:
```

Comment:
Lets include fields `controller` and `state` in status which will indicate who has modified this and what is current state like `inProgress`, `Done` etc so that this will be used by AKO and AKO gateway containers to ingest these resources.